### PR TITLE
Add CodeGuru Profiler service client

### PR DESCRIPTION
**Pull request by chlunde**
### Description

Adding service client for CodeGuru Profiler, as documented here: https://hashicorp.github.io/terraform-provider-aws/add-a-new-service/

### Relations

Relates #16768

### Output from Acceptance Testing

N/A, just the service client.

### CODE REVIEW

**Comment by chlunde** [comment]
@justinretzolk should I create the PR for the resource now (it will have to include this commit), or should I wait until this is merged?

**Review by johnsonaj — APPROVED** [review]
LGTM 🚀 

```console
% make test ./...

==> Checking that code complies with gofmt requirements...
go test ./...  -timeout=5m
?   	github.com/hashicorp/terraform-provider-aws	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/attrmap	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/acctest	3.707s
?   	github.com/hashicorp/terraform-provider-aws/internal/experimental/sync	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/framework	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/allowsubcats	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/checknames	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/customends	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/issuelabels	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/namescapslist	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/conns	4.646s
ok  	github.com/hashicorp/terraform-provider-aws/internal/create	0.426s
ok  	github.com/hashicorp/terraform-provider-aws/internal/enum	0.467s
ok  	github.com/hashicorp/terraform-provider-aws/internal/envvar	0.856s
ok  	github.com/hashicorp/terraform-provider-aws/internal/errs	0.509s
ok  	github.com/hashicorp/terraform-provider-aws/internal/errs/fwdiag	0.793s
ok  	github.com/hashicorp/terraform-provider-aws/internal/errs/sdkdiag	0.727s
ok  	github.com/hashicorp/terraform-provider-aws/internal/experimental/depgraph	0.942s
ok  	github.com/hashicorp/terraform-provider-aws/internal/flex	0.622s
ok  	github.com/hashicorp/terraform-provider-aws/internal/framework/flex	3.380s
ok  	github.com/hashicorp/terraform-provider-aws/internal/framework/types	3.134s
ok  	github.com/hashicorp/terraform-provider-aws/internal/framework/validators	1.819s
ok  	github.com/hashicorp/terraform-provider-aws/internal/generate/common	1.873s
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/prlabels	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/servicelabels	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/servicepackage	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/servicesemgrep	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/teamcity	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/logging	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/tags/templates/v2	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/generate/tags/templates/v1	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/generate/namevaluesfilters	3.626s
ok  	github.com/hashicorp/terraform-provider-aws/internal/maps	0.398s
?   	github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/provider	5.465s
ok  	github.com/hashicorp/terraform-provider-aws/internal/sdktypes	0.498s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/accessanalyzer	5.171s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/account	7.105s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/account/retry	5.983s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/acm	10.693s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/acmpca	12.569s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/amp	11.353s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/amplify	11.579s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/apigateway	5.528s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/appfabric	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/apigatewayv2	8.087s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/appautoscaling	5.385s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/appconfig	3.420s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/appflow	3.448s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/appintegrations	5.502s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/applicationinsights	7.619s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/appmesh	5.538s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/apprunner	5.543s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/appstream	3.487s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/appsync	7.679s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/athena	4.368s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/auditmanager	3.699s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/autoscaling	5.756s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/autoscalingplans	3.501s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/backup	3.605s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/batch	3.472s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/bedrock	7.390s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/budgets	5.403s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ce	3.363s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/chime	3.404s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/chimesdkmediapipelines	5.397s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/chimesdkvoice	4.627s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cleanrooms	6.658s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cloud9	8.487s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudcontrol	8.506s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudformation	8.757s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudfront	3.776s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudhsmv2	7.327s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudsearch	5.351s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/codeguruprofiler	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudtrail	4.797s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cloudwatch	5.642s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/codeartifact	3.534s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/codebuild	7.755s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/codecatalyst	9.431s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/codecommit	10.621s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/codegurureviewer	3.493s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/codepipeline	4.220s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/codestarconnections	3.594s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/codestarnotifications	3.422s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/computeoptimizer	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cognitoidentity	4.174s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cognitoidp	3.455s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/comprehend	5.545s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/configservice	7.585s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/connect	3.622s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/controltower	3.444s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/cur	5.428s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/dataexchange	5.633s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/datapipeline	7.392s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/datasync	3.469s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/dax	5.516s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/deploy	5.665s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/detective	3.526s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/devicefarm	7.709s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/directconnect	3.552s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/dlm	5.547s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/dms	3.476s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/docdb	3.628s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/docdbelastic	3.579s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ds	3.708s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/dynamodb	3.502s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ec2	21.668s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ecr	5.303s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ecrpublic	3.269s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ecs	13.591s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/efs	7.619s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/eks	9.451s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elasticache	14.743s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elasticbeanstalk	17.034s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elasticsearch	14.574s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elastictranscoder	16.368s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elb	5.224s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2	8.266s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/emr	5.899s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/emrcontainers	4.059s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/emrserverless	4.101s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/events	10.147s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/evidently	7.531s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/finspace	3.340s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/firehose	5.708s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/fis	10.501s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/fms	3.313s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/fsx	7.556s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/gamelift	5.356s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/greengrass	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/service/healthlake	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/glacier	7.542s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/globalaccelerator	3.351s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/glue	5.668s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/grafana	7.868s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/guardduty	9.867s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/iotanalytics	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/service/iotevents	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/iam	9.527s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/identitystore	9.379s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/imagebuilder	5.351s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/inspector	3.374s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/inspector2	11.104s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/internetmonitor	13.309s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/iot	11.901s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ivs	12.332s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ivschat	6.931s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/kafka	5.397s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/kafkaconnect	7.972s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/kendra	4.319s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/keyspaces	3.462s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/kinesis	3.497s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/kinesisanalytics	5.646s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/kinesisanalyticsv2	5.916s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/kinesisvideo	7.017s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/kms	3.399s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/lakeformation	5.378s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/lambda	6.798s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/lexmodels	9.008s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/lexv2models	5.530s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/mediaconnect	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/licensemanager	9.783s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/lightsail	3.401s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/location	7.605s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/logs	5.656s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/macie2	7.461s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/mediaconvert	4.086s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/medialive	3.502s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/mediapackage	5.494s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/mediastore	3.334s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/memorydb	7.400s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/meta	8.824s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/mq	10.945s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/mwaa	11.819s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/neptune	6.919s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/networkfirewall	8.946s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/networkmanager	11.210s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/oam	8.364s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/opensearch	3.687s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/opensearchserverless	3.354s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/opsworks	5.344s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/organizations	7.788s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/outposts	5.692s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/pinpoint	10.088s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/quicksight/schema	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/pipes	7.763s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/pricing	5.691s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/qldb	4.475s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/quicksight	4.653s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ram	3.454s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/rbin	5.039s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/rds	9.680s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/redshift	5.336s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/redshiftdata	3.307s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/redshiftserverless	7.445s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/resourceexplorer2	8.631s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/resourcegroups	9.957s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/resourcegroupstaggingapi	11.952s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/rolesanywhere	14.653s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/route53	10.879s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/route53domains	12.797s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/route53recoverycontrolconfig	6.933s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/route53recoveryreadiness	7.529s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/route53resolver	5.475s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/rum	3.476s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/s3	5.521s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/s3control	4.931s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/s3outposts	6.892s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/securitylake	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/sagemaker	6.402s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/scheduler	9.887s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/schemas	7.736s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/secretsmanager	9.361s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/securityhub	7.288s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/serverlessrepo	9.429s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/servicecatalog	5.389s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/servicediscovery	3.449s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/servicequotas	5.433s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ses	9.538s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/sesv2	7.829s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/sfn	3.376s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/shield	6.771s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/signer	7.879s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/simpledb	5.433s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/sns	3.668s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/sqs	7.471s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ssm	5.447s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ssmcontacts	3.432s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ssmincidents	7.537s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ssoadmin	9.261s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/storagegateway	6.563s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/sts	4.356s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/swf	9.158s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/synthetics	7.070s
?   	github.com/hashicorp/terraform-provider-aws/internal/service/verifiedpermissions	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/timestreamwrite	5.610s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/transcribe	3.530s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/transfer	8.737s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/vpclattice	3.387s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/waf	3.337s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/wafregional	4.145s
?   	github.com/hashicorp/terraform-provider-aws/internal/sweep/awsv1	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/sweep/awsv2	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/sweep/framework	[no test files]
?   	github.com/hashicorp/terraform-provider-aws/internal/sweep/sdk	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/wafv2	5.378s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/worklink	3.358s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/workspaces	7.496s
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/xray	8.076s
ok  	github.com/hashicorp/terraform-provider-aws/internal/slices	5.817s
ok  	github.com/hashicorp/terraform-provider-aws/internal/sweep	8.871s [no tests to run]
ok  	github.com/hashicorp/terraform-provider-aws/internal/tags	8.109s
?   	github.com/hashicorp/terraform-provider-aws/version	[no test files]
ok  	github.com/hashicorp/terraform-provider-aws/internal/tfresource	9.393s
ok  	github.com/hashicorp/terraform-provider-aws/internal/types	0.668s
ok  	github.com/hashicorp/terraform-provider-aws/internal/types/duration	0.696s
ok  	github.com/hashicorp/terraform-provider-aws/internal/types/nullable	0.453s
ok  	github.com/hashicorp/terraform-provider-aws/internal/types/timestamp	0.363s
ok  	github.com/hashicorp/terraform-provider-aws/internal/ujson	0.739s
ok  	github.com/hashicorp/terraform-provider-aws/internal/vault/helper/pgpkeys	1.118s
ok  	github.com/hashicorp/terraform-provider-aws/internal/vault/sdk/helper/jsonutil	0.789s
ok  	github.com/hashicorp/terraform-provider-aws/internal/verify	0.984s
ok  	github.com/hashicorp/terraform-provider-aws/names	0.334s
```

**Comment by johnsonaj** [comment]
@chlunde thank you for the contribution! 🎉

**Pull request merged**
